examples/react-dapp/src/App.tsx

```diff
@@ -1,9 +1,12 @@
 import React, { useState } from 'react'
 import './App.css'
+import './Login.css'
 import { IAM, CacheServerClient } from 'iam-client-lib'
 import axios from 'axios'
-import Button from '@material-ui/core/Button'
-import { Typography } from '@material-ui/core'
```

Comment:
If you're not using material ui, please remove it from package.json 😄

Reply:
Nice catch, thanks, I've removed it